src/aleph/netty.clj

```diff
@@ -240,19 +240,19 @@
 (defn wrap-future
   [^Future f]
   (when f
-    (if (.isSuccess f)
-      (d/success-deferred (.getNow f) nil)
-      (let [d (d/deferred nil)
-            ;; Ensure the same bindings are installed on the Netty thread (vars,
-            ;; classloader) than the thread registering the
-            ;; `operationComplete` callback.
-            bound-operation-complete (bound-fn* operation-complete)]
-        (.addListener f
-          (reify GenericFutureListener
-            (operationComplete [_ _]
-              (ensure-dynamic-classloader)
-              (bound-operation-complete f d))))
-        d))))
+    (let [d (d/deferred nil)]
+      (if (.isDone f)
+        (operation-complete f d)
+        (let [;; Ensure the same bindings are installed on the Netty thread (vars,
+              ;; classloader) than the thread registering the
+              ;; `operationComplete` callback.
+              bound-operation-complete (bound-fn* operation-complete)]
+          (.addListener f
+                        (reify GenericFutureListener
+                          (operationComplete [_ _]
+                            (ensure-dynamic-classloader)
+                            (bound-operation-complete f d))))))
```

Comment:
Oops, only now noticed that this form got reindented. However, I think this is in line with common style guides anyway :sweat_smile:

Reply:
...it's indented correctly in my code. Did it get pushed out of the correct form level somewhere else?

Reply:
On my side, I always disable all the tooling I have when working on aleph (parinfer/paredit and so on) to avoid
having this behaviour.

I'm the last person who touches this part of the code : https://github.com/clj-commons/aleph/commit/6143d5e5360e2ce00002a6ab66f2ff62eb216490#diff-ef7e90d6943fd305084d91366461caabf23c2ec88466cd4719fd66ea862417b4R244-R246

Reply:
@KingMob Ah sorry, the line I attached that comment to wasn't the most obvious. I was referring to the indentation of the second argument to the `(.addListener f ...)` call, i.e. the `(reify ...)`. It wasn't indented to the same column as `f` before.

@arnaudgeiser 
> On my side, I always disable all the tooling I have when working on aleph (parinfer/paredit and so on) to avoid having this behaviour.

Probably a good idea to keep patches minimally invasive :thinking: But preserving the idiosyncratic formatting indefinitely doesn't seem too attractive either. We discussed before to possibly introduce `cljfmt` and do a big honking re-indent commit. So I suggest to just do what seems most reasonable until then. In this case here I'd say the reindent doesn't really hurt.